server/middleware/tchannel-client.js

```diff
@@ -160,6 +160,7 @@ module.exports = async function(ctx, next) {
   ctx.cadence = {
     openWorkflows: req('ListOpenWorkflowExecutions', 'list', withDomainPaging),
     closedWorkflows: req('ListClosedWorkflowExecutions', 'list', withDomainPaging),
+    listWorkflows: req('ListWorkflowExecutions', 'list', withDomainPaging),
```

Comment:
Do the IDLs need to be updated?

Reply:
Nop, someone already updated IDL which includes the new list API